src/renderer.rs

```diff
@@ -1479,282 +1479,279 @@ impl Renderer {
+                                         1.0,
+                                         0.0)
+                                    }
+                                    LowLevelFilterOp::Blur(radius,
+                                                           AxisDirection::Vertical) => {
+                                        gl::blend_func_separate(gl::SRC_ALPHA,
+                                                                gl::ONE_MINUS_SRC_ALPHA,
+                                                                gl::ONE,
+                                                                gl::ONE);
+                                        (0.0,
+                                         radius.to_f32_px() * self.device_pixel_ratio,
+                                         0.0,
+                                         1.0)
+                                    }
+                                    LowLevelFilterOp::Contrast(amount) => {
+                                        gl::disable(gl::BLEND);
+                                        (1.0, amount.to_f32_px(), 0.0, 0.0)
+                                    }
+                                    LowLevelFilterOp::Grayscale(amount) => {
+                                        gl::disable(gl::BLEND);
+                                        (2.0, amount.to_f32_px(), 0.0, 0.0)
+                                    }
+                                    LowLevelFilterOp::HueRotate(angle) => {
+                                        gl::disable(gl::BLEND);
+                                        (3.0,
+                                         (angle as f32) / ANGLE_FLOAT_TO_FIXED,
+                                         0.0,
+                                         0.0)
+                                    }
+                                    LowLevelFilterOp::Invert(amount) => {
+                                        gl::disable(gl::BLEND);
+                                        (4.0, amount.to_f32_px(), 0.0, 0.0)
+                                    }
+                                    LowLevelFilterOp::Saturate(amount) => {
+                                        gl::disable(gl::BLEND);
+                                        (5.0, amount.to_f32_px(), 0.0, 0.0)
+                                    }
+                                    LowLevelFilterOp::Sepia(amount) => {
+                                        gl::disable(gl::BLEND);
+                                        (6.0, amount.to_f32_px(), 0.0, 0.0)
+                                    }
+                                    LowLevelFilterOp::Brightness(_) |
+                                    LowLevelFilterOp::Opacity(_) => {
+                                        // Expressible using GL blend modes, so not handled
+                                        // here.
+                                        unreachable!()
+                                    }
+                                };
```

Comment:
Yes I removed the dead code and then realised I could just do an early `continue`.

See https://github.com/servo/webrender/pull/297/files/baf8a0f94ec2acf3903e9d1bd924b1a6986c20bf..ddf0f25dca6d3b1c3031033158e5da80695d6129?w=1#r68850332.